src/main/resources/public/src/views/ProcessingStat.vue

```diff
@@ -0,0 +1,374 @@
+<template>
+    <div class="container">
+        <h2>Processing Statistics</h2>
+        <div class="form-group row">
+            <label for="claimer-stat-display-date" class="col-sm-3 col-form-label col-form-label-sm">
+                From: 
+            </label>
+            <input type="date" id="claimer-stat-display-date" v-model="datePicker"/>
+        </div>
+        <div class="form-group row">
+            <div class="general-stat">
+                <p class="p-text">DC Importer has completed 
```

Comment:
It's badly indented, and `<br/> is overused ..

Reply:
I tried to clean it a little tell me if more is needed